app/presenters/publishing_api/links_presenter.rb

```diff
@@ -70,13 +69,8 @@ def worldwide_organisation_ids
       (item.try(:worldwide_organisations) || []).map(&:content_id)
     end
 
-    def topic_content_ids
-      item.specialist_sectors.map(&:topic_content_id)
-    end
-
     def parent_content_ids
```

Comment:
Can we lose this method? Or is parent part of the links schema?

Reply:
Since parent content_ids are used elsewhere e.g. in HTML attachments I thought it might be easier to keep the method in.

Happy to give it a go at changing it. It caused a few test failures if I remember rightly which is why I thought I'd leave the empty array in.

Also please let me know if there is a smarter way of doing this 🙏🏻

Reply:
Well, HTML attachments have [their own presenter](https://github.com/alphagov/whitehall/blob/main/app/presenters/publishing_api/html_attachment_presenter.rb) that defines its own "links" object, but maybe this presenter is also used with HTML attachments. I think you'd probably know better than I would tbh

Reply:
How could I have doubted you! I just checked it again and it was the document collections I thought did some magic but I think it can be removed. Will push up the commit and double check on integration